src/utils/jsonapi.js

```diff
@@ -36,7 +36,7 @@ const serializerOptions = (serializerArgs) => {
   const resourceUrl = resourcePathLink(apiBaseUrl, resourcePath);
   const options = {
     pluralizeType: false,
-    attributes: resourceKeys,
+    attributes: _.clone(resourceKeys),
```

Comment:
this is a fix for the error where empty `relationship` keys were showing up in `included` resources. Because it was using a shallow copy of `resourceKeys`, subsequent calls to `addCompoundRelationship` in `pizzas-serializer` which mutated `attributes` were actually mutating `pizzaResourceKeys`, and the changes were persisting across API calls.

Reply:
so I think the behavior where the serializer adds an extra `relationship` key _is_ still actually a bug, it's just that we can avoid it

Reply:
nvm, changed this behavior to happen in `addCompoundRelationship` instead